### ISSUE TITLE
Regions parameter should accept an R list, not a stringified JSON dictionary

### ISSUE THREAD

**Issue by atheriel** [mountainMath/cancensus#27]
Right now the `region` parameter looks something like `region='{"PR":["59"]}'`. It would be nice to be able to use something more idiomatic to the R language, like

```r
regions = list(PR = c("59"), ...)
```

This should be possible with `jsonlite::toJSON`, although we may need to ensure that the list elements are vectors to avoid making the server barf.

**Comment by mountainMath** [comment]
Totally agree. Much better way of handling things. And jsonlite does exactly the right thing:
```
regions = list(PR = c("59"))
jsonlite::toJSON(regions)
```
yields
```
{"PR":["59"]} 
```

**Comment by atheriel** [comment]
It looks like it also does the right thing in the likely case, which is when users don't wrap arguments in `c()`:

```r
> jsonlite::toJSON(list(PR = "35", CMA = c("59155", "59255")))
# {"PR":["35"],"CMA":["59155","59255"]} 
```

**Comment by atheriel** [comment]
I'll take a crack at implementing this shortly.

**Comment by mountainMath** [comment]
Can we type-check if "regions" is a string and just continue as before in that case? Not really much extra work and the cold falls back gracefully while we transition over the function calls. We could put in a depreciation warning and phase it out in a couple of weeks.

**Comment by atheriel** [comment]
@mountainMath I've implemented this in #29.

**Comment by mountainMath** [comment]
Merged PR https://github.com/mountainMath/cancensus/pull/29
